pykrige/ok.py

```diff
@@ -404,6 +404,34 @@ def display_variogram_model(self):
                                         self.lags), 'k-')
         plt.show()
 
+    def get_variogram_points(self):
+        """Obtain the data pairs for the variogram model.
+           returns a numpy.array with the form [lag, variogram_value_at_lag]
+        """
+        variogram_points = self.variogram_function(self.variogram_model_parameters, self.lags)
+
+        return np.array([self.lags, variogram_points])

```

Comment:
I don't really see the advantage of concatenating lags and variogram points in term of the API.

Maybe just return the `variogram_points` would be sufficient? `lags` can already be accessed as an attribute.

Reply:
Well, I'll look into that, where can I read more about the API? I'd like to be in full compliance with it.

Reply:
```lags``` is a class attribute that I don't think is documented very well internally (I don't think it's mentioned in any of the docstrings...). You need the lags to do anything with the variogram points, so I actually don't mind returning them both.